vizro-core/src/vizro/models/types.py

```diff
@@ -45,6 +35,12 @@ class _SupportsCapturedCallable(Protocol):
     _captured_callable: CapturedCallable
 
 
+def validate_captured_callable(cls, value, info: ValidationInfo):
+    # TODO: We may want to double check on the mechanism of how field info is brought to
+    field_info = cls.model_fields[info.field_name]
+    return CapturedCallable._validate_captured_callable(value, field_info)
```

Comment:
```suggestion
    return CapturedCallable._validate_captured_callable(value, field_info.json_schema_extra])
```

And rewrite signatures for the validators accordingly (we could even do a `TypedDict` annotation to show the expected format of `json_schema_extra`). Unless you think this is a bad idea for some reason. Generally speaking I would pass the minimum required information between these functions.

Reply:
Good suggestion